build.yaml

```diff
@@ -9,28 +9,16 @@ jobs:
       steps:
           - template: pipeline/prerequisites.yaml
 
-          - task: Npm@1
-            inputs:
-                command: custom
-                customCommand: run tslint
-            displayName: npm run tslint
+          - script: yarn tslint
+            displayName: check lint errors
```

Comment:
Is the displayName necessary when using script? If it defaults to displaying the script being run I think that's equally readable and less duplicative

Reply:
it just displays `yarn tslint` which didn't look nice. I mean, I thought display name looked better; but thats my opinion. 
Removing `displayName` will make the yaml file a lot shorter though.

Reply:
I don't have a strong preference, whichever way you like is fine for this PR imo

Reply:
I tried without displayname in axe-sarif-converter and it actually just shows as "CmdLine", so we should definitely keep the displayNames